script/main.js

```diff
@@ -249,10 +249,23 @@ class App extends Component {
     this.setState({versions: e.target.value.split(',')});
   }
 
+  normalizeVersions(versions) {
+    return versions.map(version => {
+      if (parseInt(version) < 5) {
+        throw new Error(`Unsupported Lighthouse version (${version})`);
+      }
+      // In the future we might want a more generalized `version % 2 === 1` thing, but for now, hardcode the change.
+      if (parseInt(version) === 7) return 6..toString();
+      return version.toString();
+    }).sort().reverse();
```

Comment:
default sort sorts as strings.

Reply:
shouldnt matter tbh. leaving for later if it ever does matter :)

Reply:
BOOKMARKED